runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java

```diff
@@ -199,6 +215,12 @@ public void onGameObjectDespawned(GameObjectDespawned event)
 				//todo
 				CrystalSpike.remove(bombLocation);
 				break;
+			case 26690: //Wintertodt SnowFall
```

Comment:
This should be added to object api, and called from that.

Reply:
Sorry, bit of a noob, where would I do that?

Reply:
runelite-api...ObjectID

Reply:
So ignore the part where it tells me not to manually edit the file?
`/* This file is automatically generated. Do not edit. */`

Reply:
yes, find your object by id and import that

Reply:
it's actually under NullObjectID, as NULL_26690 🥂

Reply:
Come to think of it, it is quite odd that is missing from objectID

Reply:
But yeah, if its actually missing, just add it.

Reply:
No, don't do that, import it from NullObjectID